Parse/src/main/java/com/parse/PushService.java

```diff
@@ -103,45 +156,26 @@
     }
   }
 
-  /* package */ static void unregisterServiceLifecycleCallbacks(
-      ServiceLifecycleCallbacks callbacks) {
+  /* package */ static void unregisterServiceLifecycleCallbacks(ServiceLifecycleCallbacks callbacks) {
     synchronized (PushService.class) {
       serviceLifecycleCallbacks.remove(callbacks);
-      if (serviceLifecycleCallbacks.size() <= 0) {
```

Comment:
any reason for getting rid of this block?

Reply:
Yeah I just think it made no sense to null this. These ServiceLifecycleCallbacks are made just for internal use in tests (PushServiceTest) so we can design them as we wish, I think the old logic was way too verbose